lib/service_check.pm

```diff
@@ -57,6 +58,12 @@ our %srv_check_results = (
 );
 
 our $default_services = {
+    hpcpackage_remain => {
+        srv_pkg_name       => 'hpcpackage_remain',
+        srv_proc_name      => 'hpcpackage_remain',
+        support_ver        => $support_ver_ge15,
```

Comment:
For how to split this hpc only module, we need some solution for this:
1) give other sles test case exclude this module
2) update some logic code to exclude this module.

Reply:
I have updated logic to run hpc cases only.